app/components/curriculum_video_component/curriculum_video_component.scss

```diff
@@ -0,0 +1,82 @@
+.video-component__image {
+  padding-right: 10px;
+}
+
+.video-component__button {
+  &:visited {
+    color: $black !important;
+  }
+  &:hover {
+    color:$white !important;
+  }
+  color: $black !important;
+}
+
+.video-component-row {
+  padding-bottom: 48px;
+
+  &-content {
+    display: flex;
+
+    @include govuk-media-query($until: tablet) {
+      flex-direction: column;
+    }
+
+    &__media {
+      flex: 2;
+    }
+
+    &--details {
```

Comment:
As per comment on the other file, will need updating to reflect any change made in other file

Reply:
I'll update this to reflect a single dash